packages/valory/skills/liquidity_rebalancing_abci/skill.yaml

```diff
@@ -147,5 +147,5 @@ models:
     class_name: SharedState
 dependencies:
   open-aea-test-autonomy:
```

Comment:
I am pretty sure more than one package uses this.

Also you didn't run generators